web/src/beta/features/Assets/AssetCard/index.tsx

```diff
@@ -46,8 +46,9 @@ const Wrapper = styled.div<{ selected?: boolean }>`
   border: 2px solid ${({ selected, theme }) => (selected ? `${theme.select.main}` : "transparent")};
   padding: ${({ theme }) => theme.spacing.small}px;
   width: 100%;
-  max-width: 148px;
+  max-width: 120px;
   height: 100%;
+  max-height: 103px;
```

Comment:
Should we be doing max-width/height? Or just setting the width and height to those values?

Reply:
I think it is better to set max values , since we did that with div container